docs/source/contributing.rst

```diff
@@ -0,0 +1,30 @@
+============
```

Comment:
There is a library of good practises which is followed for linux system roles and may be useful as a reference: https://redhat-cop.github.io/automation-good-practices/

I think it may make sense to link to this as a set of good practises somewhere.